tests/migrations/test_migrations.py

```diff
@@ -299,3 +299,7 @@ def test_migration_to_ccc50ec18a7e(alembic_runner: MigrationContext, alembic_eng
 
 def test_migration_to_0040045eae14(alembic_runner: MigrationContext, alembic_engine: Engine):
     alembic_runner.migrate_up_to("0040045eae14")
+
+
+def test_migration_to_c89920be2e66(alembic_runner: MigrationContext, alembic_engine: Engine):
+    alembic_runner.migrate_up_to("c89920be2e66")
```

Comment:
these are the scripts specific to postgres as sqlite doesn't support type changes, right? I don't think we need to have just the empty tests here then, I think the imports at the top of the file cover them already:
```python
from pytest_alembic.tests import (
    test_single_head_revision,  # noqa: F401
    test_up_down_consistency,  # noqa: F401
    test_upgrade,  # noqa: F401
)
```

Reply:
Ok cool, yeah that was the purpose, just making sure the normal upgrade/downgrade alembic processes work, we don't have multiple heads, etc.